token/js/src/extensions/interestBearingMint/state.ts

```diff
@@ -6,9 +6,9 @@ import { ExtensionType, getExtensionData } from '../extensionType.js';
 
 export interface InterestBearingMintConfigState {
     rateAuthority: PublicKey;
-    initializationTimestamp: bigint;
+    initializationTimestamp: number;
     preUpdateAverageRate: number;
-    lastUpdateTimestamp: bigint;
+    lastUpdateTimestamp: number;
```

Comment:
these were changed because the buffer encoder does not work with bigints. numbers should be fine for timestamps and this doesn't seem to impact any other current usage

Reply:
Sorry, what do you mean about the buffer encoder not working? We use bigints in a lot of the extensions and they seem to work. I'd prefer to leave this as is

Reply:
Specifically if you use a bigint it as an issue when trying to encode this state due to this function that's in the buffer encoding library:
```
function divmodInt64(src) {
    const hi32 = Math.floor(src / V2E32); // cannot mix bigint and numbers here
    const lo32 = src - (hi32 * V2E32);
    return { hi32, lo32 };
}
```

Reply:
I believe to fix this we'd have to fix the buffer encoding library. I took a look through all other state type definitions I could find and didn't see any other bigints